app/controllers/application_controller.rb

```diff
@@ -90,4 +90,14 @@ def set_visitor_props
       @visitor_properties = {user: current_user.email, role: current_user.user_role}
     end
   end
+
+  def log_spam_attempt(form)
+    ip_address = request.remote_ip
+    url_orig = request.original_url
+    email = params["email"]
+    subject = params["subject"]
+    message = params["message"]
+    phony = params["phone"]
```

Comment:
I think we should change the param name to "phony" if that's what we're using for the column in the DB. It may be a little confusing down the line.

Reply:
I changed the db column to phone to be consistent. Get latest.. but you probably will need to do a 'rails db:rollback' then rerun the migration.